packages/dapp/pages/404.tsx

```diff
@@ -9,22 +9,34 @@ const Custom404 = () => {
         </p>
         <ul>
           <li>
-            <a href="//github.com/ubiquity/ubiquity-dollar/wiki" target="_blank">Docs</a>
+            <a href="//github.com/ubiquity/ubiquity-dollar/wiki" target="_blank">
+              Docs
+            </a>
```

Comment:
I've seen this formatting being reverted several times. We must have a misconfig with one and/or another formatter. Can you confirm that this was done automatically by the formatter settings? If so, I will create a new issue to fix this.

Reply:
Yes, this came up as a lint warning, saving the file changes to this format.